### ISSUE TITLE
How do I use single-text-shadow?

### ISSUE THREAD

**Issue by JamesTheHacker** [Compass/compass#2093]
I've been trying for the past 30 minutes to get `single-text-shadow` to work to no avail.

Are there any examples on correct usage? I've seen the example on the compass docs that uses default values, but I want to pass the data directly to the method.

The following doesn't appear to work:

```
@include single-text-shadow(1px, 1px, 2px, $color: black);
```

**Comment by JamesVanWaza** [comment]
@JamesTheHacker `Compass` is no longer supported use `node-sass` instead.

**Comment by mirisuzanne** [comment]
Node Sass is great, but not actually a replacement for Compass. The closest successor would be [Eyeglass](https://github.com/sass-eyeglass) — which does not include the CSS3 mixin library. I'd recommend using Auto-Prefixer instead.

**Comment by JamesVanWaza** [comment]
@mirisuzanne, Please review most of the issues and inform users that `Compass` is not being supported and offer them alternatives, there are too many open issues and no one is there to inform them.

**Comment by JamesTheHacker** [comment]
@mirisuzanne I'll try both, thanks. Why is Compass no longer active?

**Comment by JamesVanWaza** [comment]
@JamesTheHacker because better tools have emerged to do the job such as `Node-Sass`, `Susy`, `Bourbon`

**Comment by JamesTheHacker** [comment]
@jgacuca567 Makes sense. PostCSS and it's plugins are amazing! I created a CSS build tool that's able to replace my previous setup with `sass` and `compass`: [Simplex](https://github.com/JamesTheHacker/Simplex).
